w/sep15_esv.html

```diff
@@ -1,2 +1,2 @@
 <h2><center>Westminster Confession of Faith</center></h2>
-<p align="center"><a name="c21" title="c21"></a>Chapter 21: Of Religious Worship, and the Sabbath Day</p><p><a name="c21p5" title="c21p5"></a>5: The reading of the Scriptures with godly fear,<sup><a href="http://esvonline.org/ACT15:21,REV1:3">[420]</a></sup> the sound preaching<sup><a href="http://esvonline.org/2TI4:2">[421]</a></sup> and conscionable hearing of the Word, in obedience unto God, with understanding, faith and reverence,<sup><a href="http://esvonline.org/JAM1:22,ACT10:33,MAT13:19,HEB4:2,ISA66:2">[422]</a></sup> singing of psalms with grace in the heart;<sup><a href="http://esvonline.org/COL3:16,EPH5:19,JAM5:13">[423]</a></sup> as also, the due administration and worthy receiving of the sacraments instituted by Christ, are all parts of the ordinary religious worship of God:<sup><a href="http://esvonline.org/MAT28:19;1CO11:23-28,ACT2:42">[424]</a></sup> beside religious oaths,<sup><a href="http://esvonline.org/DEU6:13,NEH10:29">[425]</a></sup> vows,<sup><a href="http://esvonline.org/ISA19:21,ECC5:4-5">[426]</a></sup> solemn fastings,<sup><a href="http://esvonline.org/JOE2:12,EST4:16,MAT9:15;1CO7:5">[427]</a></sup> and thanksgivings upon special occasions,<sup><a href="http://esvonline.org/,107,EST9:22">[428]</a></sup> which are, in their several times and seasons, to be used in an holy and religious manner.<sup><a href="http://esvonline.org/HEB12:28">[429]</a></sup></p>
+<p align="center"><a name="c21" title="c21"></a>Chapter 21: Of Religious Worship, and the Sabbath Day</p><p><a name="c21p5" title="c21p5"></a>5: The reading of the Scriptures with godly fear,<sup><a href="http://esvonline.org/ACT15:21,REV1:3">[420]</a></sup> the sound preaching<sup><a href="http://esvonline.org/2TI4:2">[421]</a></sup> and conscionable hearing of the Word, in obedience unto God, with understanding, faith and reverence,<sup><a href="http://esvonline.org/JAM1:22,ACT10:33,MAT13:19,HEB4:2,ISA66:2">[422]</a></sup> singing of psalms with grace in the heart;<sup><a href="http://esvonline.org/COL3:16,EPH5:19,JAM5:13">[423]</a></sup> as also, the due administration and worthy receiving of the sacraments instituted by Christ, are all parts of the ordinary religious worship of God:<sup><a href="http://esvonline.org/MAT28:19;1CO11:23-28,ACT2:42">[424]</a></sup> beside religious oaths,<sup><a href="http://esvonline.org/DEU6:13,NEH10:29">[425]</a></sup> vows,<sup><a href="http://esvonline.org/ISA19:21,ECC5:4-5">[426]</a></sup> solemn fastings,<sup><a href="http://esvonline.org/JOE2:12,EST4:16,MAT9:15;1CO7:5">[427]</a></sup> and thanksgivings upon special occasions,<sup><a href="http://esvonline.org/EST9:22,PS107">[428]</a></sup> which are, in their several times and seasons, to be used in an holy and religious manner.<sup><a href="http://esvonline.org/HEB12:28">[429]</a></sup></p>
```

Comment:
I can't see the change on this line, but hey, if it's no change it's no problem!

Reply:
![image](https://user-images.githubusercontent.com/538363/127757888-cfade831-a19e-47ac-8b6d-54d7b579de6d.jpeg)

`icdiff` helps with isolating these partial-line diffs.

Reply:
Cool, good catch as always. It would be nicer if all this html body had reasonable linebreaks, but after this year is over and you bring the baseline to perfection, it won't be needed!